mcux/mcux-sdk/drivers/enet/fsl_enet.c

```diff
@@ -793,7 +794,8 @@ static void ENET_SetTxBufferDescriptors(enet_handle_t *handle,
     uint8_t ringNum;
     uint16_t count;
     uint32_t txBuffSizeAlign;
-    uint8_t *txBuffer                   = NULL;
+    uintptr_t txBuffer                  = 0;
```

Comment:
what is wrong with NULL?

Reply:
This update change txBuffer from an pointer to "uintptr_t" type which is uint64 for ARM64 and uint32_t for ARM32, it is to fx the size of 'buffer' in RX and TX buffer descriptors, the buffer pointer has fixed size of 4 Bytes in buffer descriptors, but the current type of 'buffer' is 'uint8_t *' , it isn't compatible with 64-bit platforms.